lib/routemaster/middleware/siphon.rb

```diff
@@ -0,0 +1,28 @@
+module Routemaster
+  module Middleware
+    class Siphon
+      # Filters out events based on their topic and passes them to a handling class
+      # Usage:
+      #    use Middleware::Siphon, 'some_topic' => SomeTopicSeriesBuilder
```

Comment:
Maybe this should be `SomeTopicHandler` as we don't know for sure it'll be a time series.